src/pages/TrackGameFormPage.js

```diff
@@ -24,10 +30,23 @@ const initialGameData = {
   isPlayersVisible: false,
   notes: ''
 };
+
 export default function TrackGameFormPage({ onTrackGame }) {
   const [step, setStep] = useState(1);
   const navigate = useNavigate();
 
+  const { response, loading, error } = useAxios({
+    method: 'get',
+    url: `/api/search?limit=100&order_by=rank&fields=id,name&client_id=${BOARDGAMEATLAS_CLIENT_ID}`
+  });
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    if (response !== null) {
+      setData(response);
+    }
+  }, [response]);
+
```

Comment:
Maybe be a bit more clear about what data this is referring to? When reading this, it seems rather generic.